tests/test_sklearn_tfidf_vectorizer_converter.py

```diff
@@ -22,22 +22,22 @@
     apply_less = None
 from skl2onnx import convert_sklearn
 from skl2onnx.common.data_types import StringTensorType, FloatTensorType
-import onnx
-import onnxruntime
+from onnxruntime import __version__ as ort_version
 from onnxruntime import InferenceSession
 from test_utils import dump_data_and_model, TARGET_OPSET
 
 
+ort_version = '.'.join(ort_version.split('.')[:2])
```

Comment:
nit: this appears multiple time, maybe put into test_utils?

Reply:
Then the code can work with onnxruntime-training as well.